app/models/miq_report/generator/html.rb

```diff
@@ -1,6 +1,6 @@
 module MiqReport::Generator::Html
   def build_html_rows(clickable_rows = false)
-    tz = get_time_zone(Time.zone.name) if Time.zone
+    get_time_zone(Time.zone.name) if Time.zone
```

Comment:
Unless `get_time_zone` changes state somewhere, this does nothing and can be removed.